src/test/java/org/jabref/gui/externalfiles/AutoSetFileLinksUtilTest.java

```diff
@@ -1,13 +1,19 @@
 package org.jabref.gui.externalfiles;
 
+import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.nio.file.StandardCopyOption;
+import java.util.ArrayList;
 import java.util.Collections;
+import java.util.LinkedList;
 import java.util.List;
+import java.util.Objects;
 import java.util.TreeSet;
 
 import javafx.collections.FXCollections;
 
+import org.jabref.AutomaticRelink;
```

Comment:
Please double check your IDE setup --> https://devdocs.jabref.org/getting-into-the-code/guidelines-for-setting-up-a-local-workspace/intellij-13-code-style.html